packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -243,6 +250,63 @@ class RenderListWheelViewport
     _hasScrolled();
   }
 
+  /// {@template flutter.rendering.wheelList.curveRate}
+  /// Affect how much the wheel is curved to the left, or the right.
+  ///
+  /// curveRate < 0 means curving to the left, curveRate > 0 means curving to
+  /// the right. The curve depends on the x-coordinate of the center of the
+  /// wheel, where that value is calculated by:
+  /// center.dx = (curveRate + 1) * size.width/2
+  /// (dx = 0 is the left border of the viewport)
+  /// For example, particularly, -1.0 means the center is on the left border
+  /// of the viewport, and 1.0 means it is on the right border.
+  ///
+  /// Defaults to 0.0, which means no curving.
+  /// curveRate should be in the range [-1.0, 1.0].
```

Comment:
Instead of saying it 'should' be in such and such range, we can just say the visual effect can be weird if it's too far from this range.

Reply:
fixed.